setup.py

```diff
@@ -23,9 +25,9 @@ def read(*rnames):
 
 setup(
     name='z3c.password',
-    version='1.1.1.dev0',
+    version='2.0.dev0',
     author="Stephan Richter, Roger Ineichen and the Zope Community",
-    author_email="zope3-dev@zope.org",
+    author_email="zope3-dev@zope.dev",
```

Comment:
```suggestion
    author_email="zope-dev@zope.dev",
```

Reply:
Thanks for catching this; one thing more to look for.